web/src/features/fba/pages/FireBehaviourAdvisoryPage.tsx

```diff
@@ -64,13 +64,11 @@ export const FireBehaviourAdvisoryPage: React.FunctionComponent = () => {
     DateTime.now().setZone('UTC-7').toISO()
```

Comment:
I know this is from another PR - but where shouldn't have -7 in some places, and 8 in others - it should all be -8 I think?